prefs.toml

```diff
@@ -31,3 +31,17 @@
    #  - "disabled" - no connection watching should be made
    #  - "print" - new connection found will be printed in the log file
    ConnectionWatcherType = "disabled"
+
+   # OverridableConfigTomlValues represents an array of items to be overloaded inside config.toml, which can be helpful
+   # when performing upgrades that bring a new config toml, and certain flags need to changed.
+   # The Path indicates what value to change, while Value represents the new value represented as string. The node operator must make sure
```

Comment:
while Value represents the new value in string format

Reply:
will be done in next release